css/css-anchor-position/anchor-position-multicol-007.html

```diff
@@ -1,33 +1,57 @@
 <!DOCTYPE html>
+<title>position-area and anchor() to all four edges works inside vertical-rl multicol</title>
 <link rel="author" title="Morten Stenshorne" href="mailto:mstensho@chromium.org">
+<link rel="author" title="Elika J. Etemad" href="http://fantasai.inkedblade.net/contact">
 <link rel="help" href="https://drafts.csswg.org/css-anchor-position-1/#anchor-pos">
 <style>
-  .query_elm {
+  .container {
+    border: solid transparent; border-width: 3px 5px 7px 9px;
+    columns: 3; column-fill: auto; gap: 10px;
+    inline-size: 230px; block-size: 100px;
+    writing-mode: vertical-rl;
+  }
+  .test {
     position: absolute;
-    left: anchor(--a1 left);
-    top: anchor(--a1 top);
-    right: anchor(--a1 right);
-    bottom: anchor(--a1 bottom);
+    position-anchor: --a1;
+    place-self: stretch;
+  }
+  .test.area {
+    position-area: center;
+  }
+  .test.func {
+    inset: anchor(inside);
+  }
+  .test.inside {
+    border: solid 5px green;
+  }
+  .test.inside.area {
+    margin: 5px;
+  }
+  .test.outside.func {
+    margin: 5px; background: green;
+    z-index: -1;
+  }
+  .test.outside.area {
+    border: 5px solid green;
+    z-index: -1;
+  }
+  .anchor {
+    anchor-name:--a1;
+    background: red;
   }
 </style>
 <link rel="match" href="../reference/ref-filled-green-100px-square.xht">
-<p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
-<div style="width:100px; height:100px; background:red;">
-  <div style="columns:3; column-fill:auto; gap:10px; inline-size:230px; block-size:100px; writing-mode:vertical-rl;">
+<p>Test passes if there is a filled green square inside each blue rectangle and <strong>no red</strong>.</p>
```

Comment:
This isn't going to match the text in ../reference/ref-filled-green-100px-square.xht
Also, there's no blue here.

Reply:
Thanks for that catch. That's leftover from some intermediary state. ^_^;